examples/default/src/screens/SettingsScreen.tsx

```diff
@@ -1,83 +1,227 @@
+      <Screen>
+        <ListTile title="Invocation Event">
+          <Select
+            label="Select Invocation Event"
+            items={[
+              {
+                label: 'None',
+                value: InvocationEvent.none,
+              },
+              {
+                label: 'Shake',
+                value: InvocationEvent.shake,
+              },
+              {
+                label: 'Screenshot',
+                value: InvocationEvent.screenshot,
+              },
+              {
+                label: 'Two fingers swipe left',
+                value: InvocationEvent.twoFingersSwipe,
+              },
+              {
+                label: 'Floating button',
+                value: InvocationEvent.floatingButton,
+                isInitial: true,
+              },
+            ]}
+            onValueChange={(value) => {
+              BugReporting.setInvocationEvents([value]);
             }}
           />
-        </InputGroup>
-      </ListTile>
+        </ListTile>
+
+        <ListTile title="Primary Color">
+          <InputGroup>
+            <InputLeftAddon>#</InputLeftAddon>
+            <Input
+              value={color}
+              maxLength={6}
+              flex={1}
+              accessibilityLabel="Primary Color Value"
+              onChangeText={(value) => {
+                setColor(value);
+                if (/^[0-9A-F]{6}$/i.test(value)) {
+                  Instabug.setPrimaryColor(`#${value}`);
+                }
+              }}
+            />
+          </InputGroup>
+        </ListTile>
+
+        <ListTile title="Theme">
+          <Select
+            label="Select Theme"
+            items={[
+              {
+                label: 'Light',
+                value: ColorTheme.light,
+              },
+              {
+                label: 'Dark',
+                value: ColorTheme.dark,
+              },
+            ]}
+            onValueChange={Instabug.setColorTheme}
+          />
+        </ListTile>
+
+        <VerticalListTile title="User Identification">
+          <VStack>
+            <View style={styles.formContainer}>
+              <View style={styles.inputWrapper}>
+                <Input
+                  placeholder="User Email"
+                  keyboardType="email-address"
+                  onChangeText={(name) => setUserEmail(name)}
+                  defaultValue={userEmail}
+                />
+              </View>
+              <View style={styles.inputWrapper}>
+                <Input
+                  placeholder=" user name"
+                  onChangeText={(name) => setUserName(name)}
+                  defaultValue={userName}
+                />
+              </View>
+              <View style={styles.inputWrapper}>
+                <Input
+                  placeholder=" user id"
+                  onChangeText={(name) => setUserID(name)}
+                  defaultValue={userID}
+                />
+              </View>
+            </View>
```

Comment:
How about we refactor these to re-usable components with the style as well as the form. This will greatly help us support other screens such as APM and App Flows, as well as using it here as it's repeated in the screen.